__tests__/app.test.js

```diff
@@ -93,5 +92,44 @@ describe('/api/topics', () => {
         
     });
 
-
-    
\ No newline at end of file
+    describe('/api/articles', () => {
+      it('GET:200 sends an array of articles object ', () => {
+          return request(app)
+          .get('/api/articles')
+          .expect(200)
+          .then((response) => {
+            expect(response.body).toHaveProperty('articles');
+            expect(Array.isArray(response.body.articles)).toBe(true);
+            expect(response.body.articles.length).toBe(13);
+            if (response.body.articles.length > 0) {
+              const article = response.body.articles[0];
+              expect(article).toHaveProperty('author');
+              expect(article).toHaveProperty('title');
+              expect(article).toHaveProperty('article_id');
+              expect(article).toHaveProperty('topic');
+              expect(article).toHaveProperty('created_at');
+              expect(article).toHaveProperty('votes');
+              expect(article).toHaveProperty('article_img_url');
+              expect(article).toHaveProperty('comment_count');
+              if (article.comment_count > 0) {
+                expect(typeof article.comment_count).toBe('number');
+            }else {
+              expect(article.comment_count).toBe(0);
+          }
+              
+          }
+          })
+        })
+      })
+      describe('GET /api/articles', () => {
+        it('Returns 404 if it is a bad request ', () => {
+            return request(app)
+            .get('/api/$Tg')
+            .expect(404)
+            .then((response)=>{
+              expect(response.statusCode).toBe(404);
+              expect(response.body).toHaveProperty('msg', "Not found")
+            })
+        });
+    });
+      
```

Comment:
We need one more test here: status 200 check articles are sorted by created_at order DESC. Jest sorted will be useful here